core/src/services/memcached/backend.rs

```diff
@@ -41,9 +41,9 @@ use crate::*;
 /// - [x] delete
 /// - [ ] copy
 /// - [ ] rename
-/// - [ ] ~~list~~
+/// - [ ] list
 /// - [ ] scan
-/// - [ ] ~~presign~~
```

Comment:
We add `~~` here to mark it's can't be implemented for this service.

Reply:
Fixed. So embarrassed 😅 LOL